components/Sidebar/index.test.jsx

```diff
@@ -0,0 +1,49 @@
+import { render, screen } from '@testing-library/react';
+
+import Sidebar, { sidebarTestIds } from '.';
+
+import useUser from 'hooks/useUser';
+import * as authenticationHelper from 'helpers/authentication';
+
+jest.mock('hooks/useUser');
+const mutateMock = jest.fn();
+
+const mockUseUser = {
+  mutateUser: mutateMock,
+};
+
+describe('Sidebar', () => {
+  const props = { name: 'john doe' };
+
+  beforeEach(() => {
+    useUser.mockImplementation(() => mockUseUser);
+
+    render(<Sidebar {...props} />);
+  });
+
+  it('renders the user name', () => {
+    const userName = screen.getByTestId(sidebarTestIds.userName);
+
+    expect(userName).toBeVisible();
+    expect(userName).toHaveTextContent(props.name);
+  });
+
+  it('renders a logout button', () => {
+    const logoutButton = screen.getByTestId(sidebarTestIds.button);
+
+    expect(logoutButton).toBeVisible();
```

Comment:
Should we test the logout label here?

```suggestion
    expect(logoutButton).toBeVisible();
    expect(logoutButton).toHvaeTextContent('logout');
```

Reply:
done in 2125544666199978509632db13f61e3cd5903fa6